pkg/apis/kubermatic/v1/external_cluster.go

```diff
@@ -68,9 +68,17 @@ type ExternalClusterSpec struct {
 
 // ExternalClusterCloudSpec mutually stores access data to a cloud provider.
 type ExternalClusterCloudSpec struct {
-	GKE *ExternalClusterGKECloudSpec `json:"gke,omitempty"`
-	EKS *ExternalClusterEKSCloudSpec `json:"eks,omitempty"`
-	AKS *ExternalClusterAKSCloudSpec `json:"aks,omitempty"`
+	GKE         *ExternalClusterGKECloudSpec     `json:"gke,omitempty"`
+	EKS         *ExternalClusterEKSCloudSpec     `json:"eks,omitempty"`
+	AKS         *ExternalClusterAKSCloudSpec     `json:"aks,omitempty"`
+	KubeOneSpec *ExternalClusterKubeOneCloudSpec `json:"kubeone,omitempty"`
```

Comment:
Why not just KubeOne? We don’t have this Spec suffix for other providers.

Reply:
done